core/storage/story/gae_models.py

```diff
@@ -95,6 +95,76 @@ def _trusted_commit(
         story_commit_log_entry.put()
 
 
+class StoryExplorationLinkModel(base_models.BaseModel):
```

Comment:
Tests are missing (but this may be foresighted if we're changing approach, see my general comment).

Reply:
Removed this and other StoryExplorationLink models and services.